sem-install

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+VERSION="0.3"
+SUPPORTED_LANGUAGES=(php ruby)
+# Misc
+GO_URL="https://storage.googleapis.com/golang"
+DATE_FORMAT='%H:%M %d/%m/%Y'
+HOST="packages.semaphoreci.com"
+
+install::check_precompiled() {
+  response=$(curl --write-out %{http_code} --head --silent --output /dev/null http://$HOST/${language}/${language_version}.tar.gz)
+  echo $response
+}
+
+install::dl_precompiled() {
```

Comment:
I would just name this function `download` it's clear enough what it will do. `dl` is too short and `precompiled` too long and not needed.

Reply:
Done